pages/common/prettier.md

```diff
@@ -22,3 +22,7 @@
 - Format files or directories recursively using single quotes and no trailing commas:
 
 `prettier --single-quote --trailing-comma {{none}} --write {{path/to/file_or_directory}}`
+
+- Format files with matching extensions:
```

Comment:
Is this in the current directory.

Reply:
`**/*.{js,jsx,ts,tsx}` contain
* `foo.js`
* `bar.ts`
* `foo/bar.js`
* `foo/bar/baz.tsx`

Match whatever is in the current directory

Reply:
I think this is an specific usage of the 4th example
```suggestion
- Format JavaScript and TypeScript files recursively, replacing the original:
```